test/webrtc.test.js

```diff
@@ -1,33 +1,275 @@
-import { WebSocket } from 'mock-socket';
```

Comment:
Do we still need this package for other tests?  If not, remove it from the project.  Otherwise, let's keep it.

Reply:
Required by socket.test.js